keyserver/handler.go

```diff
@@ -56,3 +65,12 @@ func (server *ConiksServer) handleRegistrationMessage(reg *p.RegistrationRequest
 		TB:   tb,
 	}, nil
 }
+
+func (server *ConiksServer) StoreUserPoliciesToKV(up *p.ConiksUserPolicies) error {
```

Comment:
>  if the data blob is encrypted, there is no way the server can de-serialize the data blob.

The user policies wouldn't be included in the encrypted portion, only the public key, so I think the server could still theoretically de-serialize the blob. But it would still be a privacy concern.